### PR TITLE
Fixes #6760 - Show test results in tc task

### LINKED ISSUE #6760: Show UI test results in taskcluster task
### Why/User Benefit/User Problem
Currently UI test results are returned from Firebase and only exposed as links at the end of the ui-test job log.  To access these results requires 10-15 clicks and scrolls which is not user-friendly.

### What/Requirements
We have a script that will parse the ui-test job log with a UI test summary report of passes/fails and links out to Firebase to drilldown more easily on failures.

### Acceptance Criteria (how do I know when I’m done?)
UI test reports will be logged in a markdown file and accessible via a taskcluster task (always sorted to top for easy discoverability)

### PR DESCRIPTION
**Pull request by rpappalax**
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### After merge
- [ ] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Comment by AaronMT** [comment]
@rpappalax looks like the commit missed the other additional changes and new script file

**Comment by AaronMT** [comment]
`[task 2020-04-24T21:18:02.663Z] automation/taskcluster/androidTest/ui-test.sh: line 117: ./automation/taskcluster/androidTest/parse-ui-test.py: No such file or directory
`

@rpappalax 
Whoops looks like you named the file `parse-test-test.py`.

File: automation/taskcluster/androidTest/parse-test-test.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    print(yaml.safe_dump(android_args["gcloud"]["device"]))
+
+    print("# Results\n")
+    print("| matrix | result | logs |\n")
+    print("| --- | --- | --- |\n")
+    for matrix, matrix_result in matrix_ids.items():
+        if (
+            matrix_result["outcome"] != "success"
+            # To show example out for green run (FIXME: remove)
+            or random.random() < 0.1
```
Comment (tomprince) [inline automation/taskcluster/androidTest/parse-test-test.py:64]: FIXME :wink: 
```suggestion
```

File: automation/taskcluster/androidTest/parse-ui-test.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    print(yaml.safe_dump(android_args["gcloud"]["device"]))
+
+    print("# Results\n")
+    print("| matrix | result | logs |\n")
+    print("| --- | --- | --- |\n")
+    for matrix, matrix_result in matrix_ids.items():
+        if (
+            matrix_result["outcome"] != "success"
+            # To show example out for green run (FIXME: remove)
+            or random.random() < 0.1
```
Comment (tomprince) [inline automation/taskcluster/androidTest/parse-ui-test.py:64]: This should be removed before landing.
```suggestion
```

**Comment by AaronMT** [comment]
`automation/taskcluster/androidTest/ui-test.sh: line 117: ./automation/taskcluster/androidTest/parse-ui-test.py: Permission denied`

**Comment by rpappalax** [comment]
> The changes here look fine to me, presuming they work. I have no opinions on the actual output 🙃.
We'll work on that once we get it running in a fenix branch :)

**Comment by AaronMT** [comment]
Getting there :)

Example output finally: https://github.com/mozilla-mobile/fenix/pull/10089/checks?check_run_id=634402392

I think we should remove the outcome check and just always print results, with the matrix link, etc.

**Comment by AaronMT** [comment]
https://github.com/mozilla-mobile/fenix/pull/10089/checks?check_run_id=634539120

Success results look good for now! This is a great step, we can tinker with formatting later.

**Pull request merged**